com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+# High Definition Range (HDR) Output in HDRP
+
+With HDRP it is possible to output to High Dynamic Range (HDR) devices that allows for wider color gamut and higher brightness range.
+
+HDR Output can be enabled for a given platform in **Player Settings > Other Settings > Use display in HDR mode**.  When this option is enabled, HDRP will process the frame so that when an HDR screen is used as output device it is presented correctly.
+
+## Tonemapping for HDR Output
+
+When the *Use display in HDR mode* option is set in the player settings, all the HDR related options appear in the  [Tonemapping](Post-Processing-Tonemapping.md) volume component. The available options depend on the Tonemapping mode selected.
+
+Unlike tonemapping for LDR displays, the tonemapping for HDR screens require to take into account the capabilities of the device. In particular it is important to adapt to three values:
+
+- Paper white value: determines the brightness value of a paper-white surface. In practice this will determines the overall screen brightness and what brightness the UI will map to. This latter point is important as usually unlit UI is rendered assuming that a value of 1 corresponds to a white color; this assumption is not true when it comes to HDR, so HDRP uses the paper white to tune the UI so that white UI will map to a white value on screen.
+- Minimum Brightness: the minimum brightness that the display can display.
+- Maximum Brightness: the maximum brightness that the display can display before saturating the values.
+
+All the above values are in nits (candela per square meters).
+
+While it is possible to detect the above three values from the screen outputting your content, it is possible that the values communicated by the device are going to be inaccurate. For this reason, we suggest to implement a calibration menu for your application.
+
+Specifically, detected Paper white values are often going to produce dimmer results than what the LDR content will produce, the reason is that normally TV boost brightness values when displaying LDR content, but respect the actual values when outputting HDR contents.
+
+#### Neutral
+
+![HDR-Output-Neutral](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-Neutral.png)
+
+| **Property**                         | **Description**                                              |
+| ------------------------------------ | ------------------------------------------------------------ |
+| **Neutral HDR Range Reduction Mode** | The tonemapping curve used for the Neutral tonemapper. The options are:<br />- BT2390: Uses a curve defined by the BT2390 broadcasting recommendations. (Default)<br />- Reinhard: A very simple tonemapping operator.<br /><br />This option is available only when Additional Properties are displayed. |
```

Comment:
"The curve that the Player uses for tone mapping. The options are:
* BT2390: The default. Defined by the [BT.2390](https://www.itu.int/pub/R-REP-BT.2390) broadcasting recommendations. 
* Reinhard: A very simple tonemapping operator, appropriate for situations where you prefer performance over precision."

What are the Additional Properties?

Reply:
> appropriate for situations where you prefer performance over precision."

This is not true, it has just a different look. 

>What are the Additional Properties


Advanced settings https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@12.0/manual/More-Options.html

Reply:
Ok, I was guessing on the "performance over precision", just ignore that.

Revision proposal for last sentence:
" This option is only available when you enable **[Additional Properties](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@12.0/manual/More-Options.html)**. "